gsitemap.php

```diff
@@ -546,6 +549,73 @@ protected function getCategoryLink(&$link_sitemap, $lang, &$index, &$i, $id_cate
         return true;
     }
 
+    /**
+	 * return the link elements for the manufacturer object
+	 *
+	 * @param array  $link_sitemap    contain all the links for the Google Sitemap file to be generated
+	 * @param array  $lang            language of link to add
+	 * @param int    $index           index of the current Google Sitemap file
+	 * @param int    $i               count of elements added to sitemap main array
+	 * @param int    $id_manufacturer manufacturer object identifier
+	 *
+	 * @return bool
+	 */
+	protected function getManufacturerLink(&$link_sitemap, $lang, &$index, &$i, $id_manufacturer = 0)
+	{
+		$link = new Link();
+		if (method_exists('ShopUrl', 'resetMainDomainCache')) {
+			ShopUrl::resetMainDomainCache();
+        }
+        
+        // Get manufacturers IDs
+		$manufacturers_id = Db::getInstance()->ExecuteS('SELECT m.`id_manufacturer` FROM `' . _DB_PREFIX_ . 'manufacturer` m
+			INNER JOIN `' . _DB_PREFIX_ . 'manufacturer_lang` ml on m.`id_manufacturer` = ml.`id_manufacturer`' .
+			($this->tableColumnExists(_DB_PREFIX_ . 'manufacturer_shop') ? ' INNER JOIN `' . _DB_PREFIX_ . 'manufacturer_shop` ms ON m.`id_manufacturer` = ms.`id_manufacturer` ' : '').
+			' WHERE m.`active` = 1  AND m.`id_manufacturer` >= ' . (int)$id_manufacturer.
+			($this->tableColumnExists(_DB_PREFIX_ . 'manufacturer_shop') ? ' AND ms.`id_shop` = ' . (int)$this->context->shop->id : '').
+			' AND ml.`id_lang` = ' . (int)$lang['id_lang'].
+			' ORDER BY m.`id_manufacturer` ASC'
+		);
+
+        // Process each manufacturer and add it to list of links that will be further "converted" to XML and added to the sitemap
+		foreach ($manufacturers_id as $manufacturer_id) {
+			$manufacturer = new Manufacturer((int) $manufacturer_id['id_manufacturer'], $lang['id_lang']);
+			$url = $link->getManufacturerLink($manufacturer, urlencode($manufacturer->link_rewrite), $lang['id_lang']);
+
+			$image_link = 'http'.(Configuration::get('PS_SSL_ENABLED') && Configuration::get('PS_SSL_ENABLED_EVERYWHERE') ? 's' : '').'://'.Tools::getMediaServer(_THEME_MANU_DIR_)._THEME_MANU_DIR_.((file_exists(_PS_MANU_IMG_DIR_.'/'.(int)$manufacturer->id.'-manu_default.jpg')) ? (int)$manufacturer->id.'-manu_default' : $lang['iso_code'].'-default-medium_default').'.jpg';
```

Comment:
`manu_default` doesn't exist, use `medium_default`

Reply:
Interesting.. in my production version use "manu_default", that's why I used that. Will change it to "medium_default".

Can I ask where is the file names set? (can't find it)